topdown/http.go

```diff
@@ -305,7 +335,7 @@ func createHTTPRequest(bctx BuiltinContext, obj ast.Object) (*http.Request, *htt
 		var strVal string
 
 		if s, ok := obj.Get(val).Value.(ast.String); ok {
-			strVal = string(s)
+			strVal = strings.Trim(string(s), "\"")
```

Comment:
It's OK given the existing code, but something here doesn't seem correct to me. Why are there extra `"`?

Reply:
Yea, I don't get this either; but I didn't want to change existing functionality.